mcs/build/tests.make

```diff
@@ -141,10 +143,49 @@ ifdef TEST_NUNITLITE_APP_CONFIG_RUNTIME
 	sed -i -e "/__INSERT_CUSTOM_APP_CONFIG_RUNTIME__/r $(TEST_NUNITLITE_APP_CONFIG_RUNTIME)" $(NUNITLITE_CONFIG_FILE)
 endif
 
+ifdef PLATFORM_AOT_SUFFIX
+
+DEDUP_DUMMY_CS=$(topdir)/class/lib/$(PROFILE)/DummyInflated.cs
```

Comment:
Why is it needed then (does DedupInflatedMethods.cs makes sense to you) ? Is it per library because this will break parallel build ?